server/s3_object_data_copier.cc

```diff
@@ -200,14 +200,14 @@ void S3ObjectDataCopier::write_data_block() {
   data_blocks_writing = std::move(data_blocks_read);
   // We have just separated part of data for writing and
   // also freed a 'slot' for futher reading
-
+  size_t size_of_ev_buffer = g_option_instance->get_libevent_pool_buffer_size();
```

Comment:
you can have this outside this function, 
so that it doesn't get called every time whenever function is called

Reply:
ok